### ISSUE TITLE
Paper size

### ISSUE THREAD

**Issue by ghost** [matthiasg/node-acrobat#3]
Hello,

I was wondering is it possible to provide paper size as an option, I couldn't find that information?

Kind regards

**Comment by matthiasg** [comment]
I don't think that is not supported by the adobe CLI tool. https://www.datalogics.com/products/pdf-tools/ is most likely to be the one that can provide that (though not free)
